package.json

```diff
@@ -19,10 +19,10 @@
   "author": "",
   "license": "MIT",
   "peerDependencies": {
-    "react": "^16.0.0-alpha.12",
-    "react-native": "^0.48.3"
+    "prop-types": "*",
```

Comment:
Why removing `prop-types` as dependency? Different packages can have different versions of `prop-types`, can't they?

Reply:
I have added the prop-types as an peer dependency instead. As react-native is moving away from prop-types in favour of flow typings, I think we should investigate if we can ditch proptypes for flow/typescript typings.

Reply:
I don't like `prop-types` too, but adding it as a peer dependency means that users of this lib must have `prop-types` as a direct dependency in their main `package.json`, and I don't like it. Would you mind to leave it as a direct dependency of this lib? Afterwards, I'll merge this PR immediately!

Reply:
@godness84 yeah of course! I will update it according to your feedback.  👍